docs/multisigs/committees.md

```diff
@@ -124,7 +132,7 @@ MEVBoostRelayAllowedList [`0xF95f069F9AD107938F6ba802a3da87892298610E`](https://
 
 **Snapshot:** [Proposal to form TRP Committee](https://snapshot.org/#/lido-snapshot.eth/proposal/0xc00b48275e268f26b6cebf82322f281a44acaf679f6381dd612e278174671daf)
```

Comment:
Missing line break?

Reply:
It was decided to remove line breaks in cases where a single link is specified for a forum post, a Snapshot, and an Aragon.